README.md

```diff
@@ -0,0 +1,192 @@
+Comparator compares metadata and it computes distance.
+We should test which one is better.
+
+1. ![img_2.png](images/pipeline1.png)
+2. ![img_3.png](images/pipeline2.png)
+#### Metadata creator
+MetadataCreator has:
+  - **constructor** that fills fields:
+    - size
+    - column_names
+    - column_names_clean(lowercase, only numbers and letters) 
+    - column_incomplete(if a column has more than 30 % missing values, it is marked as incomplete)
+  - **Methods for set creator**
+    - set_model: sets word transformer model
+    - compute_column_names_embeddings: computes embeddings for clean column names
+    - compute_column_kind: computes kind 
+    - compute_basic_types: compute types on top level
+    - compute_advanced_types: compute types on middle level
+    - compute_advanced_structural_types: compute types on smaller level (user should pick only one of these 3)
+    - compute_correlation: correlation between columns
+    - create_column_embeddings: create embeddings for columns
+  - *Getters*
+    - get_column_by_type: it returns all names of columns with a specified type
+    - get_numerical_columns: it returns names for all numerical columns
+    - get_metadata: it is the main method. It returns created metadata.
+
+> **Usage**:
+> firstly we call constructor, then we can call any
+> set methods (but for types we should pick just one),
+> then we can get metadata
+
+#### Comparator picture 1
+This comparator creates comparing several matrixes for each type.
+Matrix's could represent different aspects. 
+
+For example, for type int we will create:
+- a matrix comparing column names
+- a matrix comparing max values
+- a matrix comparing range
+- ...
+
+For type string we will create:
+- a matrix comparing column names
+- a matrix comparing embeddings
+- a matrix comparing the most used word
+
+Then we will create one matrix for string and one matrix for int by using
+some function to unite matrix's. 
+
+From each of these two matrixes we will compute one distance number. 
+Then these distances will be merged. 
+#### Comparator picture 2
+This comparator will create one big matrix for all columns regardless of the type. 
+Each element in the matrix will be computed from several aspects 
+(for int: column names, max value, range ...). 
+Then we create one number from this huge matrix, which is the distance
+of these two tables. 
+### Column2Vec
+Column2Vec is a module in which we implement word2Vec based functionality for columns. 
+It will compute embeddings for columns, so we can compare them. 
+More about this module can be found [here](column2Vec/README.md).
+### Types and Kinds
+We have decided to split columns by type. We can compute types or kinds for each column.
+Types define the real type of column. Some you may know from programming languages (int, float, string)
+and some are specific (human generated, word, sentence ...).
+Kinds represent higher categorization.
+
+Types have some hierarchy as you can see on picture 1. 
```

Comment:
We already had a picture 1 and picture 2

Reply:
Fixed